xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -25,6 +27,8 @@
 
 using namespace KODI::WINDOWING::GBM;
 
+const std::string SETTING_VIDEOSCREEN_LIMITGUISIZE = "videoscreen.limitguisize";
```

Comment:
can you put this in an anonymous namespace?

Reply:
this has been addressed